src/nvim/main.c

```diff
@@ -2108,6 +2108,7 @@ static void usage(void)
   mch_msg(_("  -V[N][file]           Verbose [level][file]\n"));
   mch_msg("\n");
   mch_msg(_("  --api-info            Write msgpack-encoded API metadata to stdout\n"));
+  mch_msg(_("  --clean               \"Factory defaults\" (skip user config, user plugins, shada)\n"));
```

Comment:
```suggestion
  mch_msg(_("  --clean               \"Factory defaults\" (skip user config/plugins, shada)\n"));
```

Reply:
I prefer "config and plugins", if that still fits.